commands/init.go

```diff
@@ -57,18 +92,110 @@ var initCmd = &cmds.Command{
 			}
 		}
 
-		return GetAPI(env).Daemon().Init(
-			req.Context,
-			api.RepoDir(repoDir),
-			api.GenesisFile(genesisFile),
-			api.PeerKeyFile(peerKeyFile),
-			api.WithMiner(withMiner),
-			api.DevnetTest(devnetTest),
-			api.DevnetNightly(devnetNightly),
-			api.DevnetUser(devnetUser),
-			api.AutoSealIntervalSeconds(autoSealIntervalSeconds),
-			api.DefaultAddress(defaultAddress),
-		)
+		// load configuration options
+		cfg := &DaemonInitConfig{
```

Comment:
BLOCKING: This is too much logic to live in a command. It is entirely concerned with populating the repo. recommend moving this to repo/setup.go or repo/init and then write tests for it.